packages/gnu/gcc/gcc93.spk.yaml

```diff
@@ -41,9 +41,8 @@ build:
     # no need to keep extra bloat files in info dir
     - spfs reset /spfs/share/info/**/*
   validation:
-    disabled:
-      # we are replacing the old gcc version on purpose
-      - MustNotAlterExistingFiles
+    rules:
+      - deny: AlterExistingFiles
```

Comment:
were you able to build these packages successfully? The old rule was disabling the validation, so in the new setup you would need `- allow: AlterExistingFiles` in order to achieve the same behavior